packages/core/src/payment/strategies/klarnav2/klarnav2-payment-strategy.spec.ts

```diff
@@ -159,7 +153,7 @@ describe('KlarnaV2PaymentStrategy', () => {
             strategy.execute(payload);
             expect(klarnaPayments.authorize)
                 .toHaveBeenCalledWith({ payment_method_category: paymentMethod.id }, getKlarnaV2UpdateSessionParamsPhone(), expect.any(Function));
-            expect(paymentMethodActionCreator.loadPaymentMethod).toHaveBeenCalledWith(paymentMethod.gateway);
```

Comment:
Because you replace this method with loadPaymentMethod from PaymentMethodRequestSender, the expectation is to verify if that method have been called, you also need to create the mock.